install_tools.sh

```diff
@@ -153,3 +153,24 @@ else
   echo "installed Dotfiles successfully."
   touch $INSTALLED/dotfiles
 fi
+
+#VCPKG
+USER_DOCUMENTS=/home/$USER/Documents
```

Comment:
Would it be possible to install vcpkg in `~/Documents/sources/`? Then it would be more consistent with the rest of the installed libraries and tools.